### PR TITLE
OpenOCD 0.11.0

### PR DESCRIPTION
**Pull request by paulfertser**
- [n] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [n] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [n] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [n] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Sorry, no macOS system handy, I hope I can count on CI for this one.

### CODE REVIEW

File: Formula/open-ocd.rb [hunk 1]
```diff
@@ -11,7 +11,6 @@ class OpenOcd < Formula
   end
 
   bottle do
-    rebuild 2
```
Comment (carlocab) [inline Formula/open-ocd.rb:14]: Please don't touch the `bottle` block. Squash your commits together when you revert this. Thanks!

**Review by carlocab — APPROVED** [review]

**Comment by paulfertser** [comment]
Hi Carlo,

Thank you for the review.

On Mon, Mar 08, 2021 at 05:36:01AM -0800, Carlo Cabrera wrote:
>    bottle do
> -    rebuild 2
> 
> Please don't touch the bottle block.

TBH, that's not at all obvious from the docs. I figured the
maintainers would update the hash sums, it's mentioned in few places
(I tried doing my homework and read few documents related to the
bottles) but nothing was said about the rebuild number and the
description said it was optional, so for a new version I felt like
it'd be natural to remove it. Please consider clarifying the docs in
this regard.

> Squash your commits together when you revert this.

My "github pull request" has just one commit, nothing to squash
there. I guess you meant to ask me to provide just a single commit for
merging after fixing the problem you highlighted, but that comes
naturally because it's easier to commit --amend rather than to create
a new commit and then squash them so this remark was a bit confusing
to me.

**Comment by carlocab** [comment]
The note not to touch the `bottle` block is here in step 4: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#create-your-pull-request-from-a-new-branch

Admittedly, it could be more prominent.

>so this remark was a bit confusing
to me

Would it be less confusing if I mention that in 95 out of 100 pull requests I review, any suggestions I make are added as new commits, rather than amended into old ones?

**Comment by paulfertser** [comment]
On Mon, Mar 08, 2021 at 06:05:38AM -0800, Carlo Cabrera wrote:
> The note not to touch the bottle block is here in step 4:
> [1]https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#create-your-pull-request-from-a-new-branch

Indeed, I missed that part because I saw it explaining some basic
things about how to use git and github, mentioning specific commands
etc, so wasn't reading attentively enough, sorry.

>   so this remark was a bit confusing
>   to me
> 
> Would it be less confusing if I mention that in 95 out of 100 pull requests I
> review, any suggestions I make are added as new commits, rather than amended
> into old ones?

Hm, probably CI can be made to special-case something like "pull
request has more than one commit each modifying the same single file"
to be an error if it's that common.

Thank you for the review and caring about the project!

**Pull request closed without merging**